pkg/sql/information_schema.go

```diff
@@ -262,6 +263,29 @@ https://www.postgresql.org/docs/9.5/infoschema-applicable-roles.html`,
 	},
 }
 
+var informationSchemaCharacterSets = virtualSchemaTable{
+	comment: `character sets available in the current database
+` + docs.URL("information-schema.html#character_sets") + `
+https://www.postgresql.org/docs/9.5/infoschema-character-sets.html`,
+	schema: vtable.InformationSchemaCharacterSets,
+	populate: func(ctx context.Context, p *planner, dbContext *dbdesc.Immutable, addRow func(...tree.Datum) error) error {
+		return forEachTableDesc(ctx, p, dbContext, hideVirtual, func(
+			db *dbdesc.Immutable, scName string, table catalog.TableDescriptor,
+		) error {
+			return addRow(
+				tree.DNull,                 // character_set_catalog
+				tree.DNull,                 // character_set_schema
+				tree.NewDString("UTF8"),    // character_set_name: UTF8 is the only available encoding
+				tree.NewDString("UCS"),     // character_repertoire: UCS for UTF8 encoding
+				tree.NewDString("UTF8"),    // form_of_use: same as the database encoding
+				tree.DNull,                 // default_collate_catalog
```

Comment:
i think this one should be the current database name.

Reply:
According to this [conversation](https://github.com/cockroachdb/cockroach/issues/8675#issuecomment-728349838), it should be `NULL`

Reply:
Fixed